### PR TITLE
fix(docs): remove inline dev inquiries

### PR DESCRIPTION
**Pull request by jonhnet**
* clean up my [inline questions] that should have been resolved in the original conversation
* fixed a relative path link to features/libraries that was broken

### CODE REVIEW

**Comment by jonhnet** [comment]
@jrasm91 review+label por favor

**Review by jrasm91 — APPROVED** [review]

File: docs/docs/guides/external-library.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # External Library
 
-This guide walks you through adding an [External Library](../features/libraries#external-libraries).
+This guide walks you through adding an [External Library](../../features/libraries#external-libraries).
```
Comment (jrasm91) [inline docs/docs/guides/external-library.md:3]: I don't think this is right, why did you change it?

Comment (jonhnet) [reply docs/docs/guides/external-library.md:3]: It seems wrong to me too, but look at the rendered version of the current version. The link doesn't go where it should.

**Comment by jonhnet** [comment]
I've removed the relative path problem from this PR; some deeper mystery needs plumbing. Now this PR is uncontroversial.
